src/PayID/pay-id-client.ts

```diff
@@ -43,6 +52,140 @@ export default class PayIDClient {
    * @param useHttps Whether to use HTTPS when making PayID requests. Most users should set this to 'true' to avoid
    *                 Man-in-the-Middle attacks. Exposed as an option for testing purposes. Defaults to true.
    */
+  constructor(network: string, useHttps: boolean = true) {
+    this.wrappedPayIdClient = new PayIdClient(network, useHttps)
+  }
+
+  /**
+   * Retrieve the address associated with a PayID.
+   *
+   * @param payID The payID to resolve for an address.
+   * @returns An address representing the given PayID.
+   */
+  async addressForPayID(payID: string): Promise<CryptoAddressDetails> {
+    return this.wrappedPayIdClient.addressForPayId(payID)
+  }
+
+  /**
+   * Generate a new invoice with compliance requests.
+   *
+   * @param payID The Pay ID to request an invoice for.
+   * @param nonce A randomly selected nonce that is unique to this invoice request.
+   */
+  async getInvoice(
+    payID: string,
+    nonce: string,
+  ): Promise<SignatureWrapperInvoice> {
+    return this.wrappedPayIdClient.getInvoice(payID, nonce)
+  }
+
+  /**
+   * Post an invoice with compliance data.
```

Comment:
Are we still using invoice now?

Reply:
AFAIK we are but I could be out of the loop. 

I queried the PayID folks and I'll leave this PR open until I get a definitive answer.

Reply:
This is `paymentSetupDetails` now... but Travel Rule is pretty faked and not real, so do what you will.